### PR TITLE
Update variant page in silico predictor display

### LINKED ISSUE #1345: Add negative warning/danger thresholds for Pangolin
Kathryn Russell reported that we didn't give a warning/danger thresholds for negative Pangolin scores (for splice loss). 

Could we add a second set of thresholds for Pangolin? 

image
```
  pangolin_largest_ds: { label: 'Pangolin', warningThreshold: 0.2, dangerThreshold: 0.5 },

```
To add: 
```
  pangolin_largest_ds: { label: 'Pangolin', warningThreshold: -0.2, dangerThreshold: -0.5 },

```

### PR DESCRIPTION
**Pull request by rileyhgrant**
Resolves #1345

- Re-adds a note referring users to SpliceAI Lookup that was mistakenly dropped after being included in mockups
- Updates Pangolin scores warning and danger thresholds to be based off of the absolute value of the score so negative (splice loss) predictions get correctly colored

Example localhost link to variant [http://localhost:8008/variant/7-66991136-C-T?dataset=gnomad_r4](http://localhost:8008/variant/7-66991136-C-T?dataset=gnomad_r4)

### CODE REVIEW

File: browser/src/VariantPage/VariantInSilicoPredictors.tsx [hunk 1]
```diff
@@ -93,7 +118,9 @@ const VariantInSilicoPredictors = ({ variant, datasetId }: Props) => {
               const predictor = predictors[id]
 
               let color = null
-              const parsedValue = parseFloat(value)
+              const parsedValue = predictor.absoluteValue
```
Comment (sjahl) [inline browser/src/VariantPage/VariantInSilicoPredictors.tsx:121]: i think using the absolute value here is probably reasonable here, as long as we know the thresholds are always going to be the same on either side of 0.

i.e., if the need to specify:

`pangolin_largest_ds: { label: 'Pangolin', warningThreshold: 0.2, dangerThreshold: 0.5 },`

and

`pangolin_largest_ds: { label: 'Pangolin', warningThreshold: -0.3, dangerThreshold: 0.6 },`

where the thresholds are different numbers for positive and negative, we might need something different here.

But, disregard if those will always match 👍

**Review by sjahl — APPROVED** [review]
Looks good to me. Just the one comment on absolute value inline, but merge away if that's not a problem

File: browser/src/VariantPage/VariantInSilicoPredictors.tsx [hunk 2]
```diff
@@ -19,13 +20,43 @@ const FLAG_DESCRIPTIONS = {
...
+    dangerThreshold: 0,
+    absoluteValue: false,
+  },
+  polyphen_max: {
+    label: 'PolyPhen (max)',
+    warningThreshold: 0.978,
+    dangerThreshold: 0.999,
+    absoluteValue: false,
+  },
+  cadd: { label: 'CADD', warningThreshold: 25.3, dangerThreshold: 28.1, absoluteValue: false },
```
[the remark this answers is not in the document]
Comment (rileyhgrant) [reply browser/src/VariantPage/VariantInSilicoPredictors.tsx:59]: Yeah this irked me a bit too lol. Oh well.

File: browser/src/VariantPage/VariantInSilicoPredictors.tsx (hunk 1 above)
Comment (rileyhgrant) [reply browser/src/VariantPage/VariantInSilicoPredictors.tsx:121]: Yeah good eye, this is the first one that even needed negative value thresholds. For this instance negative denotes a different class of prediction (splice gained vs splice loss) that shares the same thresholds. Without knowing I'd hope its a fair guess that other predictors may follow a similar pattern if they allow negative scores?

We can revisit later down the line if needed.

**Comment by KoalaQin** [comment]
Thanks! I don't see the change yet, it's after the deploy?

**Pull request merged**